src/WooCommerce.php

```diff
@@ -539,8 +539,19 @@ public static function woocommerce_get_item_data($data, $cartItem) {
    */
   public static function woocommerce_display_item_meta($html, $item, $args) {
     $strings = [];
+    $data = [];
     $product = $item->get_product();
-    $product_data_set = array_merge(static::getProductData($product), static::getProductAttributes($product));
+    // Extract key/value pairs from passed html and add it to the product data set.
+    $matched = preg_match_all('@<strong.+[^>]>(.+)</strong>.+<p>(.+)</p>@', $html, $matches, PREG_SET_ORDER);
+    if ($matched) {
+      foreach ($matches as $match) {
+        $data[] = [
+          'name' => rtrim($match[1], ':'),
+          'value' => $match[2],
```

Comment:
Would it be easier to extract and remove all order item meta data values within the `woocommerce_order_item_meta` hook (or similar name) and temporarily attach them on each `$item`, so that the incoming `$html` is empty and you can output the original/raw meta data here instead of parsing it from the HTML?

Reply:
I first thought of this as well, but don't know why I did not do it in the end 🙈. Added a fixup commit, thanks!